packages/camera/camera/ios/Classes/CameraPermissionUtils.m

```diff
@@ -0,0 +1,39 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import AVFoundation;
+#import "CameraPermissionUtils.h"
+
+void FLTRequestCameraPermissionWithCompletionHandler(
+    FLTCameraPermissionRequestCompletionHandler handler) {
+  switch ([AVCaptureDevice authorizationStatusForMediaType:AVMediaTypeVideo]) {
+    case AVAuthorizationStatusAuthorized:
+      handler(nil);
+      break;
+    case AVAuthorizationStatusDenied:
+      handler([FlutterError errorWithCode:@"CameraAccessDeniedWithoutPrompt"
```

Comment:
Do we need to add documentation to the README for these errors?

https://github.com/flutter/plugins/blob/08e4e02ef2a104e591fcaa92f8ff1f57775f75af/packages/camera/camera/README.md?plain=1#L79

What about the example app?
https://github.com/flutter/plugins/blob/08e4e02ef2a104e591fcaa92f8ff1f57775f75af/packages/camera/camera/example/lib/main.dart

Reply:
Updated example and README. Also checked that the short code snippet in the README file still works.